nautobot/extras/views.py

```diff
@@ -20,7 +20,7 @@
 from nautobot.core.views import generic
 from nautobot.dcim.models import Device
 from nautobot.dcim.tables import DeviceTable
-from nautobot.extras.utils import get_worker_count
+from nautobot.extras import utils as extras_utils
```

Comment:
Why this change? Looks like you're still only using get_worker_count.

Reply:
Yes, but to be able to patch it I cannot import it directly, we have to import the module. Otherwise the patch will not be picked up.

Reply:
Does it work if you import it directly and change the patch to `patch(nautobot.extras.views.get_worker_count)`?

Reply:
Good call, I changed it to that!